pkg/azdiskschedulerextender/azdiskschedulerextender.go

```diff
@@ -432,6 +420,8 @@ func setNodeScoresToZero(nodes []v1.Node) (priorityList schedulerapi.HostPriorit
 	return
 }
 
+// diskRequestedByPod == total number of v2 disks requested by pod
+// totalRequestedDiskCount = total number of pvc disks requested by pod
 func getQualifyingDisksRequestedByPod(ns string, requestedVolumes []v1.Volume, filterOutAccessModes ...v1.PersistentVolumeAccessMode) (disksRequestedByPod map[string]struct{}, totalRequestedDiskCount int) {
```

Comment:
is `totalRequestedDiskCount` used anywhere now? I think we can probably remove it.